scripts/coverage/merge_coverage_files.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# Copyright (C) 2024 Intel Corporation
+# Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+[ "$1" != "" ] && PREFIX="$1" || PREFIX="exports-coverage"
```

Comment:
pls add a short comment on top, explaining the script and params

Reply:
Done.